src/coreComponents/python/CMakeLists.txt

```diff
@@ -1,61 +1,68 @@
-set( ENV{LD_LIBRARY_PATH}, ENV{LD_LIBRARY_PATH}:${PYTHON_DIR}/lib/ )
+# set( ENV{LD_LIBRARY_PATH}, ENV{LD_LIBRARY_PATH}:${PYTHON_DIR}/lib/ )
 
-set( PYTHON_INTERP_INCLUDE_DIRS
-     ${PYTHON_DIR}/include/python2.7/
-     ${PYTHON_DIR}/lib/python2.7/site-packages/numpy/core/include/
-   )
+# set( PYTHON_INTERP_INCLUDE_DIRS
+#      ${PYTHON_DIR}/include/python2.7/
+#      ${PYTHON_DIR}/lib/python2.7/site-packages/numpy/core/include/
+#    )
 
-# Set python style targets
-# set( python_sources modules/geosx_mesh_tools_package/geosx_mesh_tools/abaqus_converter.py )
+# blt_import_library( NAME python_interp
+#                     INCLUDES    ${PYTHON_INTERP_INCLUDE_DIRS}
+#                     HEADERS     ${PYTHON_INTERP_INCLUDE_DIRS}
+#                     HEADERS_OUTPUT_SUBDIR python_interp
+#                     LIBRARIES python2.7 
+#                   )
```

Comment:
I think this is very old leftover code when we were first testing python integration in the code